dapp/src/pages/AllDao.vue

```diff
@@ -1,80 +1,91 @@
 <template>
-	<div class="margin_bottom_sm">
-		<h3 class="text_center">Sigma DAOs</h3>
-		<div class="flexbox_justify_space">
-			<a-input-search
-				style="width: 40%; margin: auto"
-				enter-button
-				placeholder="Search DAO by Name"
-				v-model:value="searchText"
-				@search="handleFilterData()"
-			/>
-			<a-button
-				type="primary"
-				:disabled="!walletStore.address"
-				@click="handleCreateDAO"
-			>
-				Create a DAO
-				<template #icon><PlusOutlined /></template>
-			</a-button>
-		</div>
+	<div v-if="showNetworkSelection">
```

Comment:
You can move this to App.vue file and check if `v-if=walletStore.network !== NetworkTypes.NONE`

Reply:
done, updated the screenshot also